pax/simulation.py

```diff
@@ -482,8 +506,8 @@ def s1_photons(self, n_photons, recoil_type, x=0., y=0., z=0, t=0.):
             # Hence, we don't care about primary & secondary excimers at all:
             timings = self.singlet_triplet_delays(
                 np.zeros(n_photons),
-                t1=self.config['alpha_singlet_lifetime_liquid'],
-                t3=self.config['alpha_triplet_lifetime_liquid'],
+                t1=self.config['singlet_lifetime_liquid'],
```

Comment:
Because alpha_** wasn't there in the .ini. And NEST model simply assumes the same singlet/triplet lifetime for ER and alpha. So just modified to make alpha mode work as Marco requested.